### PR TITLE
Add Balancer/Aura

### PR DESCRIPTION
**Pull request by nick**

### CODE REVIEW

File: sections/homepage/Allocation.tsx
Comment (rafaelugolini) [inline sections/homepage/Allocation.tsx:1]: I noticed you changed the current code styling, and I also noticed that we don't have a prettier configuration in this repo.

File: sections/homepage/Hero.tsx [hunk 1]
```diff
@@ -161,10 +161,10 @@ const Hero = ({ apy, tvl, tvlUsd, sectionOverrideCss }: HeroProps) => {
           <HeroInfo
             title="Trusted yield sources"
             subtitle="Only battle-tested, blue-chip protocols are used to generate OETH's market-leading, risk-adjusted yield."
-            className="w-full md:w-1/2 bg-origin-bg-dgrey rounded-lg mr-0 md:mr-7 mb-6 md:mb-0"
+            className="w-full md:w-1/2 bg-origin-bg-dgrey rounded-lg mr-0 md:mr-7 mb-6 md:mb-0 text-wrap-balance"
           >
             <Image
-              src={assetRootPath("/images/yield-sources.svg")}
+              src={assetRootPath("/images/yield-sources.png")}
```
Comment (rafaelugolini) [inline sections/homepage/Hero.tsx:167]: I remember we had issues with the png getting low-res

File: sections/homepage/Allocation.tsx
Comment (nick) [reply sections/homepage/Allocation.tsx:1]: I added a prettier config and re-formatted to use the existing style

File: constants/protocolMapping.ts [hunk 2]
```diff
@@ -11,6 +11,12 @@ const protocolMapping = {
     description:
       "Lido is a liquid staking solution for Ethereum that staking ETH without locking tokens or maintaining infrastructure. OETH holds stETH to earn staking rewards for participating in the Ethereum network and adds a layer of auto-compounding.",
   },
+  Aura: {
+    color: "#FFFFFF",
+    image: "/images/aura-logo.png",
```
Comment (rafaelugolini) [inline constants/protocolMapping.ts:16]: should we use SVG there?

**Review by rafaelugolini — APPROVED** [review]

**Comment by rafaelugolini** [comment]
@nick you will need to sign your commits

**Pull request merged**